project-backend/Models/DAOs/WorkerDAO.cs

```diff
@@ -0,0 +1,17 @@
+﻿using project_backend.Models.User;
+using System;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace project_backend.Models.Worker
+{
+    [Table("workers")]
+    public class WorkerDAO
+    {
+        [Key]
+        public int UserId { get; set; }
+        public UserDAO User { get; set; }
+        public DateTime WorkerSince { get; set; } = DateTime.Now;
```

Comment:
This is a temporary implementation until more details are provided for the worker. We'll change it when doing the apply for worker feature.